plugins/globalias/globalias.plugin.zsh

```diff
@@ -0,0 +1,11 @@
+globalias() {
+   zle _expand_alias
+   zle expand-word
+   zle self-insert
```

Comment:
Oh right, if this isn't there then it doesn't add the extra space when there's nothing to expand. Forget about it.